XRTK-Core/Packages/com.xrtk.core/Inspectors/Profiles/InputSimulationSystem/MixedRealityInputSimulationSystemProfileInspector.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using XRTK.Definitions.InputSimulationSystem;
+using XRTK.Inspectors.Utilities;
+using XRTK.Services.InputSimulationSystem;
+
+namespace XRTK.Inspectors.Profiles.InputSimulationSystem
```

Comment:
Can we change the namespace to 
```
XRTK.Inspectors.Profiles.InputSystem.Simulation
```
Makes more sense that way, as the input system is required for simulation, it's not a replacement